fml/status_or.h

```diff
@@ -0,0 +1,66 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FML_STATUS_OR_H_
+#define FLUTTER_FML_STATUS_OR_H_
+
+#include <optional>
+
+#include "flutter/fml/status.h"
+
+namespace fml {
+
+/// TODO(): Replace with absl::StatusOr.
```

Comment:
In place (or in addition to) a todo, it would be nice to document or link to documents on how this should be used.

Reply:
done